locale/de.json

```diff
@@ -30,21 +30,7 @@
 		"article-body-block.interstitial-link-aria-label": "Ähnlichen Artikel öffnen"
 	},
 	"full-author-bio-block": {
-		"full-author-bio-block.connect-text": "Verbinden",
-		"full-author-bio-block.social-email": "E-Mail",
-		"full-author-bio-block.social-twitter": "Auf Twitter verbinden",
-		"full-author-bio-block.social-facebook": "Auf Facebook verbinden",
-		"full-author-bio-block.social-instagram": "Auf Instagram folgen",
-		"full-author-bio-block.social-rss": "RSS Feed",
-		"full-author-bio-block.social-linkedin": "Auf LinkedIn verbinden",
-		"full-author-bio-block.social-reddit": "Auf Reddit verbinden",
-		"full-author-bio-block.social-youtube": "Auf YouTube verbinden",
-		"full-author-bio-block.social-medium": "Auf Medium verbinden",
```

Comment:
can you delete this from lokalise as well? It looks like I can see it in lokalise. want to make sure we're accounting for changes so that the global phrases in lokalise can have this info. Let me know if you want help working with lokalise

![Screen Shot 2022-08-30 at 15 07 52](https://user-images.githubusercontent.com/5950956/187533245-7bc65c62-b2f8-4120-9e12-a4ec3237453e.png)

Reply:
I guess migrate them

Reply:
we want to make sure we keep the translations